### PR TITLE
new story

### PR DESCRIPTION
**Pull request by KikeMendez**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2326)

### CODE REVIEW

**Comment by KikeMendez** [comment]
new story in spanish

**Comment by cbuckey-uda** [comment]
Sorry about the delay, @KikeMendez! @casey-collab, please review this.

File: espanol/oso/oso.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+El oso edgardo estaba tranquilo en su cueva cuando de pronto:
```
Comment (casey-collab) [inline espanol/oso/oso.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@KikeMendez, I think maybe there was a misunderstanding. Rather than adding a new story, you should continue the existing story.

**Comment by KikeMendez** [comment]
Ups so sorry. I'll try to fix this, Thanks, and thanks to teach me Git.. Its helping me a lot.:)

**Comment by cbuckey-uda** [comment]
@KikeMendez, you're welcome! I'm glad you're enjoying the course.

@casey-collab, please take another look

File: espanol/oso/oso.md (hunk 1 above)
Comment (casey-collab) [inline espanol/oso/oso.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@KikeMendez, I don't speak Spanish, so I'm not sure, but based on your title, it looks like maybe you are trying to add a new story? The purpose of this repository is to contribute to the existing story. That's why Casey is complaining that the story is not "reachable" - she wants you to add on to the current story rather than making something that stands on its own.

**Comment by KikeMendez** [comment]
Hello:
I have deleted the "oso" story, and I've added a new one.. I hope this fix the conflict..

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :dog:

**Comment by KikeMendez** [comment]
:) awesome! I'm so happy.  I have a question, where can I keep learning git? Are you thinking of doing another course? Can you suggest any place where I can learn more about it?
Again thank you so much for everything girls. Excellent course!

**Pull request merged**